packages/paste-style-props/src/types/css-props.ts

```diff
@@ -0,0 +1,21 @@
+import type {Properties} from 'csstype';
+
+export type AnimationProperty = Properties['animation'];
+export type AppearanceProperty = Properties['appearance'];
+export type BoxSizingProperty = Properties['boxSizing'];
+export type ClipProperty = Properties['clip']; // 'clip' is deprecated
```

Comment:
I can remove this comment, looks like technically it's deprecated, but there is a special "bucket" on the interface for deprecated types so... 🤷‍♀️

Reply:
I think it's fine 🤷🏻‍♂️